Makefile

```diff
@@ -112,6 +112,15 @@ lint-fix: ## Fix linting issues automagically
 #	@go run -modfile tools/go.mod ./internal/lint -fix $$(go list ./... | grep -v '/acceptance/')
 	@go run -modfile tools/go.mod github.com/daixiang0/gci write -s standard -s default -s "prefix(github.com/hacbs-contract/ec-cli)" .
 
+.PHONY: tekton-lint
+tekton-lint: ## Check for tekton-lint, if not found provide link. Otherwise, execute tekton-lint for all yaml in the tasks subdirectory, it's smart enough to ignore non-Tekton yaml files. Any warnings are currently considered failures.
+    ifdef $(GITHUB_ACTIONS)
```

Comment:
Is this check inverted? It would be helpful to do it when running locally, and not on GitHub. (Or maybe I'm misreading this.)

Reply:
Fixed!